src/components/common/ColonyActions/ActionDetailsPage/DefaultMotion/MotionPhaseWidget/StakingWidget/StakingInput/StakingSliderMessages/RequiredStakeMessage.tsx

```diff
@@ -4,7 +4,9 @@ import { defineMessages, FormattedMessage } from 'react-intl';
 import classNames from 'classnames';
 
 import Numeral from '~shared/Numeral';
-import { getStakeFromSlider, getStakePercentage } from '../..';
+import { STAKING_THRESHOLD } from '~constants';
```

Comment:
I just made a comment in #344 about the staking threshold being a constant.

I guess this answers my concern :P